.docker/worker.dockerfile

```diff
@@ -37,6 +37,8 @@ RUN apk add --no-cache \
         zip \
     && apk del --purge .build-deps
 
+RUN apk add --no-cache exiftool
```

Comment:
The worker image is now based on Debian so you have to use `apt-get`.

Reply:
resolved 3577945dd93a1a6e008ae38b899f284479e024fa